tests/console/yast2_tftp.pm

```diff
@@ -70,6 +70,10 @@ sub run {
         send_key 'alt-o';    # close the window
         assert_screen 'yast2_tftp_open_port';    # assert that window is closed
     }
+    # bsc#1207390, skip open firewall ports for tumbleweed
+    elsif (match_has_tag('yast2_tftp_no_network_interfaces') && is_tumbleweed) {
```

Comment:
the cheapest operation first (but it is just my assumption that all those needle-related operations has more expensive, perhaps are just a search in a map after a previous opencv procedure):
 (is_tumbleweed && match_has_tag('yast2_tftp_no_network_interfaces') but just in case of doubt...